repl/src/main/scala_2.10/org/apache/spark/repl/h2o/H2OInterpreter.scala

```diff
@@ -50,7 +50,15 @@ class H2OInterpreter(sparkContext: SparkContext, sessionId: Int) extends BaseH2O
       logWarning("ADD_JARS environment variable is deprecated, use --jar spark submit argument instead")
     }
     val jars = {
-      val userJars = Utils.getUserJars(conf, isShell = true)
+      val userJars = {
```

Comment:
@goosalex I think it would be cleaner to use reflection at this stage to call corresponding methods then specifying some list of dependencies (this might change without failing), also this does not mimic the exact Spark behaviour.

What I think would be the best to try calling `getLocalUserJarsForShell(conf)` if the method exists we can call it and can continue, if it doesn't we can fallback to Utils.getUserJars. This way, the internal implementation of these methods remains hidden, but we can work on both Spark 2.1.1 and 2.1.0 at the same time.

What do you think ?